### ISSUE TITLE
Eva Leu's PhD phytoplankton data

### ISSUE THREAD

**Issue by cnrdh** [npolar/marine-db#47]
Two data files contributed for publication by Eva Leu in 2019, see #46.

**Comment by AnetteWold** [comment]
If this only contain metadata I think we should skip it. I don't have any result data from Eva Leu and I think part of the data is experiemtnal data (if I remember correctly)

**Comment by cnrdh** [comment]
No, there should be real data some of it quite easy to include (search in Google docs for her name)

**Comment by AnetteWold** [comment]
Ok,hva må gjøres?

**Comment by cnrdh** [comment]
Fant du noe? Send i såfall URLer (2-3?)

**Comment by AnetteWold** [comment]
Phytoplankton2004 Kb1/Kb2/kb3: https://docs.google.com/spreadsheets/d/18NgllNF4l5GumtEFl6dWEh-L7WNO1v-GE_d2cdLftBE/edit?usp=sharing

Phytoplankton-2003-Eva_Leu-PhD: https://docs.google.com/spreadsheets/d/1sV2yAJ6IjOh4A87wZh_QrATu_03GmZwb7eTrpH71Qyc/edit?usp=sharing

Anette

**Comment by cnrdh** [comment]
Great, as you see, they are almost done as DwC
